pnc-model/src/main/java/org/jboss/pnc/model/BuildConfiguration.java

```diff
@@ -54,9 +56,11 @@
     @ManyToOne(cascade = { CascadeType.MERGE, CascadeType.PERSIST, CascadeType.REFRESH })
     private ProductVersion productVersion;
```

Comment:
@pgier  Just looking at the model, didn't we removed ProductVersion from BuildConfiguration? And have it only inside BuildConfiguratioSet?